app.js

```diff
@@ -23,3 +23,24 @@ app.get('/', (req, res) => {
 });
 
 app.listen(3000, () => console.log('🏃‍ on port 3000'));
```

Comment:
Be sure to check which port you're using and clean your app.js when you do. Was there a specific reason why your port 3000 wasn't working?